taxonomies/categories.txt

```diff
@@ -38219,7 +38240,7 @@ fr:Spaghetti, spaghettis
 hu:Spagetti
```

Comment:
Issue here with lemon soy yoghurts and spaghetti.

nl:Citroensojayoghurt, Citroensojayoghurts, Sojayoghurt met citroen